charts/gardener-extension-os-gardenlinux/templates/service.yaml

```diff
@@ -7,10 +7,6 @@ metadata:
     networking.resources.gardener.cloud/from-all-seed-scrape-targets-allowed-ports: '[{"port":{{ .Values.metrics.port }},"protocol":"TCP"}]'
     networking.resources.gardener.cloud/namespace-selectors: '[{"matchLabels":{"kubernetes.io/metadata.name":"garden"}}]'
     networking.resources.gardener.cloud/pod-label-selector-namespace-alias: extensions
-    # TODO: This label approach is deprecated and no longer needed in the future. Remove them as soon as gardener/gardener@v1.75 has been released.
-    networking.resources.gardener.cloud/from-policy-pod-label-selector: all-seed-scrape-targets
-    networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.metrics.port }},"protocol":"TCP"}]'
-    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
```

Comment:
I think `networking.resources.gardener.cloud/from-world-to-ports` has to be preserved. 
It was added in https://github.com/gardener/gardener-extension-os-gardenlinux/commit/03d5ce33c6323961993c28b0fa9d8b7d54b07911#diff-9b15882254880940d89bb390fceb1a41c85a56f1ee5438cbf97349d680e13d08R13

If you check for label key in Gardener for the first 2 labels, you won't find anything. However, the `networking.resources.gardener.cloud/from-world-to-ports` label key is still valid and supported. See https://github.com/gardener/gardener/blob/63cb5ea67f7e3882851d76db111816ed32b04ec7/pkg/apis/resources/v1alpha1/types.go#L173-L175